docs/products/clickhouse/concepts/data-integration-overview.md

```diff
@@ -0,0 +1,112 @@
+
+## Managed credentials integration
+
+The managed credentials integration uses the
+[ClickHouse named collections](https://clickhouse.com/docs/en/operations/named-collections)
+logic. It allows storing key-value pairs that are to be used as credentials when
+connecting to external data sources. With the managed credentials integration enabled,
+querying data is easier and quicker since you no longer need connections parameters in each
+query. They are stored and available from in-Aiven credential storage.
+
+On top of integrating credentials, the managed credentials integration allows integrating
+data from external sources. For that purpose, you create tables using table engines.
+
+Managed credentials integration in Aiven for ClickHouse is supported with the following
+data source types:
+
+- PostgreSQL®
+- MySQL®
+- Amazon S3®
+
+:::note[See also]
+For information on how table engines work in Aiven for ClickHouse services, preview
+[Engines: database and table](/docs/products/clickhouse/concepts/service-architecture#engines-database-and-table).
+For the list of table engines available in Aiven for ClickHouse, check
+[Supported table engines](/docs/products/clickhouse/reference/supported-table-engines).
+:::
+
+## Managed databases integration
+
+The managed databases integration allows using a database engine for handling your
+external data. When enabled, this type of integration provides you with an automatically
+created database, where you can have your data ingested.
+
+Managed databases integration in Aiven for ClickHouse is supported with the following
+data source types:
+
+- PostgreSQL®
+- Apache Kafka®
+
+:::note[See also]
+For information on how database engines work in Aiven for ClickHouse services, preview
+[Engines: database and table](/docs/products/clickhouse/concepts/service-architecture#engines-database-and-table).
+For more information on ClickHouse database engines, see
+[Database engines](https://clickhouse.com/docs/en/engines/database-engines).
+:::
+
+## Supported data source types
+
+Depending on a data source type, Aiven for ClickHouse supports different integration modes.
+
+| Data source type | Data source integration<br/>(with Aiven service <br/>or external source)| Managed databases integration| Managed credentials integration |
+|------------------|----------------------------|------------------------------|---------------------------------|
+| PostgreSQL       | <ConsoleIcon name="tick"/> | <ConsoleIcon name="tick"/>   | <ConsoleIcon name="tick"/>      |
+| MySQL            | <ConsoleIcon name="tick"/> | <ConsoleIcon name="cross"/>  | <ConsoleIcon name="tick"/>      |
+| Apache Kafka     | <ConsoleIcon name="tick"/> | <ConsoleIcon name="tick"/>   | <ConsoleIcon name="cross"/>     |
+| ClickHouse       | <ConsoleIcon name="tick"/> | <ConsoleIcon name="cross"/>  | <ConsoleIcon name="cross"/>     |
```

Comment:
ClickHouse has managed credentials I think?  @tilman-aiven

Reply:
ClickHouse to ClickHouse -> managed credentials only 👍